modules/tinymce/src/themes/silver/main/ts/api/Options.ts

```diff
@@ -274,7 +275,7 @@ const isReadOnly = option('readonly');
 const getHeightOption = option('height');
 const getWidthOption = option('width');
 const getMinWidthOption = wrapOptional(option('min_width'));
-const getMinHeightOption = wrapOptional(option('min_height'));
+const getMinHeightOption = option('min_height');
```

Comment:
Why was this changed from being wrapped in an `Optional`? Instead you're now duplication the logic by having to use `Optional.from` everywhere. Please look at reverting this unless there's a good reason it was changed.

Reply:
I don't have a good reason beside trying to be consistent as other options that have a default value. Happy to revert it because it looks less messy

Reply:
I'm not sure what you mean here by consistent as everything around it does the same wrapping sorry 🤷 Anyways, thanks for putting this back 👍